templates/project/Project__c__.xml

```diff
@@ -1,8 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Sorry, additional comment! I don't think this is intended to have been deleted! Now when you create a project it's missing the project file!